com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineMaterialUpgrader.cs

```diff
@@ -391,9 +392,22 @@ public static void UpdateStandardSpecularMaterialKeywords(Material material)
             CoreUtils.SetKeyword(material, "_METALLICSPECGLOSSMAP", material.GetTexture("_SpecGlossMap"));
             CoreUtils.SetKeyword(material, "_SPECULAR_SETUP", true);
             UpdateSurfaceTypeAndBlendMode(material);
+            UpdateDetailScaleOffset(material);
             BaseShaderGUI.SetupMaterialBlendMode(material);
         }
 
+        static void UpdateDetailScaleOffset(Material material)
+        {
+            // In URP details tile/offset is tied to base map tile offset
+            var baseScale = material.GetTextureScale("_BaseMap");
```

Comment:
Is this specific to our shaders? _BaseMap is attributed as [MainTexture] for us and default is of course _MainTexture.

Reply:
It is specific from builtin->urp shader where tile/offset needs to be recalculated to retain same visual